pandas/_testing.py

```diff
@@ -1140,6 +1141,8 @@ def assert_series_equal(
         Whether to compare internal Categorical exactly.
     check_category_order : bool, default True
         Whether to compare category order of internal Categoricals.
+    check_freq : bool, default True
```

Comment:
move to below versionadded:: 1.0.2 for check_category_order and add versionadded:: 1.1.0